cogboard-webapp/src/components/Widget/index.js

```diff
@@ -43,6 +43,7 @@ const Widget = ({ id, index }) => {
   const showUpdateTime = widgetTypes[type]
     ? widgetTypes[type].showUpdateTime
     : false;
+  const updateTimestamp = getWidgetUpdateTime(content);
```

Comment:
I can wrap showUpdateTime and updateTimestamp in one function managing both things - please let me know which approach you consider more  readable / clear

Reply:
One function would be more concise and probably more clear.

Reply:
You can wrap functionality related to widget's update time field in a hook to cut unnecessary logic out of the component. There is more pieces like this as a future candidates to refactor.